src/map/battle.c

```diff
@@ -1568,10 +1568,10 @@ int battle_calc_skillratio(int attack_type, struct block_list *src, struct block
 					skillratio += 10 * skill_lv - 30;
 					break;
 				case SL_STIN:
-					skillratio += (tst->size!=SZ_SMALL?-99:10*skill_lv); //target size must be small (0) for full damage.
+					skillratio += (tst->size!=MOBSIZE_SMALL?-99:10*skill_lv); //target size must be small (0) for full damage.
 					break;
 				case SL_STUN:
-					skillratio += (tst->size!=SZ_BIG?5*skill_lv:-99); //Full damage is dealt on small/medium targets
+					skillratio += (tst->size!=MOBSIZE_BIG?5*skill_lv:-99); //Full damage is dealt on small/medium targets
```

Comment:
honestly I have no idea why estun can deal full damage to small size monster
at least according to client translation, estun should only deal full damage to medium size monster